### ISSUE TITLE
[Accessibility] tree items should be visible in 400% zoom

### ISSUE THREAD

**Issue by setaskin** [microsoft/vscode#120759]
- VS Code Version: 1.55.0 stable 
- OS Version: Windows 

Steps to Reproduce:

1. set to screen display resolution to 1280 x 1024
2. go to vs code explorer page
3. drag the tree separator all the way to the right in maximum width
4. increase zoom to 400%
Actual  : some items in the tree view are not completely visible. Low Vision users will not be able to see what is on the screen. 
Expected : in 400% zoom all items should be visible. 

![400](https://user-images.githubusercontent.com/37163639/113931416-494c4580-97a7-11eb-898f-dbe816f2a773.gif)

Does this issue occur when all extensions are disabled?: Yes

**Comment by setaskin** [comment]
@isidorn , this is another bug that is filed by accessibility team on us that I think coming from VS Code.

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/115878
